README.md

```diff
@@ -125,6 +125,9 @@ There are two flavors of live ISO images:
 
 You can download them from the [openSUSE Build
 Service](https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Devel/images/iso/).
+* Make sure to download the correct ISO file according to your system architecture(Example: You would need to choose {https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Devel/images/iso/agama-live.x86_64-7.0.0-openSUSE-Build1.17.iso} if your processor is based on x64 architecture.
```

Comment:
I got your point here, but I do not think it's a good idea linking to a specific version of Agama in the README. @imobachgs, @mvidner, do you have any idea how to improve this?

Reply:
There is always an ISO build which does not contain the build number:, e.g.: https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Staging/images/iso/agama-live.x86_64-openSUSE.iso

However, I'd not recommend using it. It makes bug reporting more difficult as you do not know the exact build number you used. And you can also accidentally use a wrong file without noticing.

It is an example so I think it is fine.

Reply:
Well, it is actually a redirection. See:

```
$ curl -i https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Staging/images/iso/agama-live.x86_64-openSUSE.iso
HTTP/2 302 
date: Thu, 11 Jan 2024 13:03:28 GMT
server: Mojolicious (Perl)
cache-control: public, max-age=300
content-length: 0
location: https://downloadcontent.opensuse.org/repositories/systemsmanagement:/Agama:/Staging/images/iso/agama-live.x86_64-7.0.0-openSUSE-Build2.62.iso
vary: Accept,COUNTRY,X-COUNTRY,Fastly-SSL
content-type: application/x-iso9660-image
```

Using a browser (tested with Firefox and Chromium) you get a file with the correct name. For cURL, you might need to do:

```
curl -L -O  https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Staging/images/iso/agama-live.x86_64-openSUSE.iso
```

For wget you need to do:

```
wget --turst-server-names https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Staging/images/iso/agama-live.x86_64-openSUSE.iso
```

Anyway, I agree with @lslezak, it is just an example.